com.github.ImCompressor.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "com.github.ImCompressor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "imcompressor",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
is there a way you could reduce the permissions scope here? I *think* not given the need for drag and drop, but just checking.

Reply:
I've used the "--filesystem=home" permission because of mozjpeg and optipng. Without this and with using the Gtk Native File Chooser it didn't works.